### PR TITLE
Convert to ESM, fix warning, remove unused files

### PR DESCRIPTION
**Pull request by rygine**
# Summary

* Added `type: "module"` to `package.json`
  * Updated `tsconfig.json` for Cypress
* Removed Storybook tooling, configs, and stories  
* Removed unused Docker files
* Updated `caniuse-lite` DB
* Fixed `vite` warning
* Upgraded `vite` and `cypress` dependencies

### CODE REVIEW

File: .github/workflows/fmt-check.yml [hunk 1]
```diff
@@ -13,5 +13,5 @@ jobs:
           cache: "npm"
       - name: Run Code Format Check
         run: |
-          npm install 
+          npm install --force
```
Comment (daria-github) [inline .github/workflows/fmt-check.yml:16]: why is this needed now?

**Review by daria-github — APPROVED** [review]

File: .github/workflows/fmt-check.yml (hunk 1 above)
Comment (rygine) [reply .github/workflows/fmt-check.yml:16]: i upgraded `vite` and the rewrite all plugin we use says it's not compatible with it so the install fails. it is compatible.
